scipy/optimize/_linprog_highs.py

```diff
@@ -281,8 +281,9 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
 
             mip_gap : float
                 The difference between the final objective function value
-                and the final dual bound. Only present when `integrality`
-                is not `None`.
+                and the final dual bound, scaled by the final objective 
+                function value. Only present when `integrality` is not 
+                `None`.
```

Comment:
The MIP gap that HiGHS returns is a relative mip_gap. Someone from the HiGHS team can double check me on this, but based on some testing as well as this [line](https://github.com/ERGO-Code/HiGHS/blob/c0e1182bd350d4ecfc4600aee3e7ff71aacae6cd/src/lp_data/HighsInfo.h#L261) in the HiGHS code which explicitly identifies this as a %, I think this is correct.

Reply:
OK, we should wait for confirmation on this because `milp` says "The difference between the final objective function value and the final dual bound." (not the _relative_ difference). It's surprising to me that the option name include "`rel`" but the return attribute's name doesn't if they are both relative.

Reply:
Sure. Here are the results of a quick investigation (code [here](https://gist.github.com/amlimaye/feb06a22fa4d642c70d49ead0efbd059)). I solve the problem in `test_mip6` repeatedly, except each time multiply `c` by a scale factor (hence scaling the objective `c^T x` by a constant factor). 

If the mip_gap returned was an absolute gap, it should itself scale by the scale factor (for a fixed mip_rel_gap = 0.01). 

If it is a relative gap, it should be the same for each value of the scale factor, and match up with a manual calculation of (primal_bound - dual_bound) / primal bound. Indeed, I see the latter:

```
aditya@olivegrove highs-sandbox % python3 mip_rel_gap_scaling_behavior.py
-----
scale_factor = 1.0, mip_rel_gap (requested) = 0.01, objective = 1853.9999999999984
mip_abs_gap = 17.99999999999841, mip_rel_gap (computed) = 0.009708737864076819, mip_rel_gap (reported) = 0.009708737864076819
-----
scale_factor = 10.0, mip_rel_gap (requested) = 0.01, objective = 18539.999999999985
mip_abs_gap = 179.99999999998545, mip_rel_gap (computed) = 0.009708737864076892, mip_rel_gap (reported) = 0.009708737864076892
-----
scale_factor = 100.0, mip_rel_gap (requested) = 0.01, objective = 185399.99999999985
mip_abs_gap = 1799.9999999998545, mip_rel_gap (computed) = 0.009708737864076892, mip_rel_gap (reported) = 0.009708737864076892
```

Reply:
@jajhall, tagging you here as you are a HiGHS contributor who commented on this PR earlier. Could you please confirm/direct us to someone from the HiGHS team who would be able to confirm this behavior?

Thanks!